### PR TITLE
BAN-138: Add story with different form components to zen-form-group

### PR DESCRIPTION
**Pull request by f-jauregui**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/BAN-138)

#### Description
As per JIRA ticket

#### ChangeLOG

- [ ] I have added all notable deployment/development environment (onprem!) changes to [CHANGELOG.md](https://github.com/reciprocity/zen-ui/blob/main/README.md)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: src/components/zen-form-group/zen-form-group.stories.mdx [hunk 1]
```diff
@@ -75,6 +109,19 @@ Wrapper around input that ensures correct styles and spacings
   </Story>
 </Canvas>
 
+### 3. Joining different form groups
```
Comment (mplesec) [inline src/components/zen-form-group/zen-form-group.stories.mdx:112]: NIT: Maybe go with "Composing" instead

**Review by mplesec — APPROVED** [review]
👍

**Review by chavlji — APPROVED** [review]
It looks nice.

But this wasn't the purpose of this ticket 😂 Title says:
*"Add stories for Form group* **for each form component** *"*
Nada requested it some month ago. She said we should create form-group story inside each component that supports it. So in the stories of input, textarea,...

I'm merging this one and cloning it to a new ticket:
https://reciprocitylabs.atlassian.net/browse/BAN-168

**Comment by f-jauregui** [comment]
> It looks nice.
> 
> But this wasn't the purpose of this ticket 😂 Title says:
> _"Add stories for Form group_ **for each form component** _"_
> Nada requested it some month ago. She said we should create form-group story inside each component that supports it, so you don't have to go to form-group stories to see the example. So in the stories of input, textarea,...
> 
> I'm merging this one and cloning it to a new ticket:
> https://reciprocitylabs.atlassian.net/browse/BAN-168

🤣 got a bit confused by the title.
@chavlji don't worry, no need to clone the ticket again.
Let me do it in this PR to make it quicker and still solving the issue.

**Pull request merged**